lib/msal-common/src/url/UrlString.ts

```diff
@@ -192,6 +192,21 @@ export class UrlString {
         return Constants.EMPTY_STRING;
     }
 
+    /**
+     * Parses query string from given string. Returns empty string if no query symbol is found.
+     * @param queryString
+     */
+    static parseQueryString(queryString: string): string {
+        const queryIndex1 = queryString.indexOf("?");
```

Comment:
Possible to use `URL` and/or `URLSearchParams`? https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams/URLSearchParams#examples

Reply:
The response doesn't come in as a fully formed URL so that API throws